src/ixdat/plotters/ms_plotter.py

```diff
@@ -266,35 +356,77 @@ def plot_vs(
                     tspan_bg=tspan_bg,
                     remove_background=remove_background,
                     include_endpoints=False,
+                    return_quantity=use_quantity,
+                )
+
+            if ureg("mol / s / cm ** 2").check(unit):
```

Comment:
Is there a way to do somethign like
`if "length^{-2}" in unit.dimensions: ...`
? 
Then there wouldn't have to be a repeat of this block for quantified vs non-quantified.